src/main/webapp/pages/accessionvalidationrange/accessionValidationRange.jsp

```diff
@@ -35,6 +37,10 @@
 	String searchTerm = request.getParameter("searchTerm");
 	String url = request.getAttribute("javax.servlet.forward.servlet_path").toString();	
 	//boolean showTestSectionSelect = !ConfigurationProperties.getInstance().isPropertyValueEqual(Property.configurationName, "CI RetroCI");
+	List<? extends Number> highRangeHighResult = ResultService.getHighRangeHigh();
```

Comment:
it is generally discouraged to add more scriptlets <% %> into the JSPs as it is an old style of doing things. Note below about converting this to use JSTL and mirroring the existing validation logic